### PR TITLE
Update documentation "Using Subsites in combination with Fluent" part

### PR DESCRIPTION
**Pull request by MLindenhofer**
References: #509 - the yml-config solution for setting the Fluent locale instead of the Subsite-locale is not existing anymore. I replaced the old solution with a workaround which works for me. 

it's my first PR :)

### CODE REVIEW

File: README.md
Comment (ChloeSartorelli) [inline README.md:1]: Please revert the unrelated style changes you have made to this file. Those changes aren't related to the purpose of this pull request.

File: README.md [hunk 1]
```diff
@@ -220,11 +220,16 @@ or by defining the subsiteCMSShowInMenu function in your admin:
...
 
-When using Subsites in combination with Fluent module, the Subsites module sets the i18n locale to the language defined in the current Subsite. When this behaviour is not desired and you need to use the locale in FluentState use the following setting in your yml config file:
+When using Subsites in combination with Fluent module, the Subsites module sets the i18n locale to the language defined in the current Subsite. When this behaviour is not desired and you need to use the locale in FluentState you can set the current Fluent locale in your PageController like this:
 
-```yaml
-    SilverStripe\Subsites\Extensions\SiteTreeSubsites:
-      ignore_subsite_locale: true
+*PageController.php*
+```
+:::php
```
Comment (ChloeSartorelli) [inline README.md]:
```suggestion
```php
```

File: README.md [hunk 2]
```diff
@@ -220,11 +220,16 @@ or by defining the subsiteCMSShowInMenu function in your admin:
...
+     protected function init(){
```
Comment (ChloeSartorelli) [inline README.md]:
```suggestion
     protected function init()
     {
```

File: README.md [hunk 3]
```diff
@@ -220,11 +220,16 @@ or by defining the subsiteCMSShowInMenu function in your admin:
...
+        if ($this->dataRecord->SubsiteID !== 0){
+            i18n::set_locale(FluentState::singleton()->getLocale());
+        }
+     }
 ```
```
Comment (ChloeSartorelli) [inline README.md]: Can you please explain why you're making this change? What was wrong with the yaml configuration documented here?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Hi,

Thank you for your contribution, and sorry it's taken so long to get back to you. There are some requested changes and clarification questions below.

File: README.md (hunk 3 above)
Comment (MLindenhofer) [reply README.md]: I make this change, because the yaml configuration documented is not in use in the Subsite-Module. So even with the yml configuration set, the i18n-Locale is still the one defined for the subsite and not the one from fluent. My solution is just one possible workaround.

File: README.md
Comment (MLindenhofer) [reply README.md:1]: Done :)

File: README.md (hunk 2 above)
Comment (ChloeSartorelli) [reply README.md]: Please make this change, so that the documentation is showing best practice.

File: README.md (hunk 3 above)
Comment (ChloeSartorelli) [reply README.md]: > I make this change, because the yaml configuration documented is not in use in the Subsite-Module. 

That yaml configuration is used here:
https://github.com/silverstripe/silverstripe-subsites/blob/4e292b8e89f9ee214340a215285cdcb526a2b5f8/src/Extensions/SiteTreeSubsites.php#L399-L417

Comment (MLindenhofer) [reply README.md]: ah I see - okay. In module version 2.6.0 it wasn't there :). Seems it changed in the meanwhile. 
Than my change is unnecessary of course .

Comment (ChloeSartorelli) [reply README.md]: Okay cool. I'll close this PR then. Thank you for your efforts and patience though, it's definitely appreciated.
